src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -0,0 +1,297 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void SingleVariableDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int P1 { get; set; }
+    static void Main(string[] args)
+    {
+        /*<bind>*/int i1;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: System.Int32 i1 (OperationKind.VariableDeclaration)
+";
+            VerifyOperationTreeForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void MultipleDeclarations()
+        {
+            string source = @"
+class Program
+{
+    static void Main(string[] args)
+    {
+        /*<bind>*/int i1, i2;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: System.Int32 i1 (OperationKind.VariableDeclaration)
+  IVariableDeclaration: System.Int32 i2 (OperationKind.VariableDeclaration)
+";
+            VerifyOperationTreeForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void FixedStatementDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int i1;
+    static void Main(string[] args)
+    {
+        var reference = new Program();
+        unsafe
+        {
+            fixed (/*<bind>*/int* p = &reference.i1/*</bind>*/)
+            {
+
+            }
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IFixedStatement (OperationKind.FixedStatement)
```

Comment:
I created issues for each thing affected in this PR, and an uber issue for tracking all of these: https://github.com/dotnet/roslyn/issues/18064